action.yaml

```diff
@@ -23,6 +23,10 @@ inputs:
   mcc_project:
     description: "MACH Composer Project"
     required: true
+  allow_component_not_found:
+    description: "Allow component not found in MCC. This will cause the component to always be marked as changed."
+    required: false
+    default: "false"
```

Comment:
shouldn't the default just be true?
I won't break any of the existing functionality I suppose